Dockerfile

```diff
@@ -58,6 +58,8 @@ RUN apt-get --yes update \
   libc++-${llvm_version}-dev \
   libfl2 \
   libgmp-dev \
+  # `virt-make-fs` from `libguestfs-tools` is needed for creating qcow2 images.
+  libguestfs-tools \
```

Comment:
FYI this implies that you need to rebuild and push the docker image via

- https://github.com/project-oak/oak/blob/main/scripts/docker_build
- https://github.com/project-oak/oak/blob/main/scripts/docker_push

But let me know if the push fails, we may have to add you to some other group

Reply:
+1. I rebuilt the image locally for testing, but thought I should wait to push the image until the pull request is merged. Is that the recommended workflow?

Reply:
You can push any time; once you do that, it will automatically update the image digest in the other scripts, so that it does not interfere with other developers until this is actually merged. i.e. the published docker image is actually pinned by digest, not just "latest", so you don't need to coordinate with others

Reply:
Note that docker_build will update the image id, and docker_push will update the image digest, and we need both to be in sync when submitted (confusing, I know, but that's how Docker works)

Reply:
Got it, thanks for explaining the workflow!
Just attempted scripts/docker_push and got a permissions error:

```
denied: Permission "artifactregistry.repositories.uploadArtifacts" denied on resource "projects/oak-ci/locations/europe-west2/repositories/oak-development"
```

How should I proceed in this case?

Reply:
I just granted you the uploadartifacts permission. Could you try again and let me know if it still does not work in like 5 mins?

Reply:
It worked, thank you! Just uploaded a commit that includes the updates from running docker_build and docker_push.